Cannot find name VoidFunction

I’m trying to use `uvu` in a nodejs project, but I get a typescript error that `VoidFunction` is not defined.

```
node_modules/uvu/index.d.ts:16:10 - error TS2304: Cannot find name 'VoidFunction'.  
```

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -13,7 +13,7 @@ declare namespace uvu {
 		skip(name?: string, test?: Callback<T>): void;
 		before: Hook<T>;
 		after: Hook<T>
-		run(): VoidFunction;
+		run(): () => void;
 	}
 }
 

